exercises/sum-of-multiples/.meta/generator/sum_of_multiples_case.rb

```diff
@@ -0,0 +1,8 @@
+require 'generator/exercise_case'
+
+class SumOfMultiplesCase < Generator::ExerciseCase
+  def workload
+    indent_lines(["assert_equal #{expected}, SumOfMultiples.new" \
+                  "(#{factors.join(', ')}).to(#{limit})"] , 4)
```

Comment:
Hmm interesting. Thanks.

The whole assertion helper thing is quite new and has many unexplored edge cases.

Creating an issue about this might be a good idea.